src/xdocs/config_coding.xml

```diff
@@ -3588,7 +3588,31 @@ for (String line: lines) {
         <source>
 &lt;module name=&quot;NestedTryDepth&quot;/&gt;
         </source>
-
+        <p>
+            Example of violation:
+        </p>
+        <source>
+try {
+    try {
+        try { // violation: Max default allowed depth is 1
+        } catch (Exception e) {
+        }
+    } catch (Exception e) {
+    }
+} catch (Exception e) {
+}
+        </source>
+        <p>
+            Example of correct code:
+        </p>
+        <source>
+try {
+    try { // OK
```

Comment:
Since you word this as "correct code", what about if people do need all the try/catches and can't remove one? Your examples are not real world cases and thus may lead to some confusion on how they should correct a violation.

Can we change the catches so they are all unique exceptions, and then show how they should correct the code if they need all the catches. In this scenario the only way would be to split the code into a new method.

They could alternatively use a Java 7 feature to have multiple exceptions caught in 1 block. I would not limit your examples to just 1 case, but show various different ones.

Reply:
> should I also add cases like

no need.

Reply:
> Can we change the catches so they are all unique exceptions

it is ok to keep single `Exception` for all, Check do not care of that is caught, it just counts `try`.